samples/copy-from-container/pom.xml

```diff
@@ -0,0 +1,87 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
I believe that we need to add `copy-from-container` module into "samples/pom.xml" POM (into the `modules` section), don't we?